### ISSUE TITLE
error when building the Key/Value server artifacts (v0.11.0)

### ISSUE THREAD

**Issue by k-o-ta** [privacysandbox/protected-auction-key-value-service#19]
I tried building the Key/Value server artifacts following the [doc](https://github.com/privacysandbox/fledge-key-value-service/blob/main/docs/deploying_on_aws.md), and I got an error below.
```
% production/packaging/aws/build_and_test --with-ami us-east-1                                                                                                                            
==== Sourcing builder.sh =====                                                                                                                                                                                                                                                                                                                                                                                                   
==== Running build_and_test_all_in_docker =====                                                                                                                                                                                                                                                                                                                                                                                  
fix end of files...........................................................Passed                                                                                                                                                                                                                                                                                                                                                
fix utf-8 byte order marker................................................Passed                                                                                                                                                                                                                                                                                                                                                
mixed line ending..........................................................Passed                                                                                                                                                                                                                                                                                                                                                
trim trailing whitespace...................................................Passed                                                                                                                                                                                                                                                                                                                                                
check for case conflicts...................................................Passed                                                                                                                                                                                                                                                                                                                                                
check for merge conflicts..................................................Passed                                                                                                                                                                                                                                                                                                                                                
check yaml.................................................................Passed                                                                                                                                                                                                                                                                                                                                                
check json.................................................................Passed                                                                                                                                                                                                                                                                                                                                                
check for broken symlinks..................................................Passed                                                                                                                                                                                                                                                                                                                                                
check for added large files................................................Passed                                                                                                                                                                                                                                                                                                                                                
check vcs permalinks.......................................................Passed                                                                                                                                                                                                                                                                                                                                                
check that executables have shebangs.......................................Passed                                                                                                                                                                                                                                                                                                                                                
detect private key.........................................................Passed                                                                                                                                                                                                                                                                                                                                                
Executable shell script omits the filename extension.......................Passed                                                                                                                                                                                                                                                                                                                                                
Non-executable shell script filename ends in .sh...........................Passed                                                                                                                                                                                                                                                                                                                                                
Check file encoding........................................................Passed                                                                                                                                                                                                                                                                                                                                                
Test shell scripts with shellcheck.........................................Passed                                                                                                                                                                                                                                                                                                                                                
buf format.................................................................Passed                                                                                                                                                                                                                                                                                                                                                
clang-format...............................................................Passed                                                                                                                                                                                                                                                                                                                                                
addlicense.................................................................Passed                                                                                                                                                                                                                                                                                                                                                
addlicense check...........................................................Passed                                                                                                                                                                                                                                                                                                                                                
terraform fmt..............................................................Passed                                                                                                                                                                                                                                                                                                                                                
prettier...................................................................Passed                                                                                                                                                                                                                                                                                                                                                
lint markdown..............................................................Passed                                                                                                                                                                                                                                                                                                                                                
buildifier.................................................................Passed                                                                                                                                                                                                                                                                                                                                                
cpplint....................................................................Passed                                                                                                                                                                                                                                                                                                                                                
black python formatter.....................................................Passed
=== cbuild debian action envs ===                                                                                                                                                                                                                                                                                                                                                                                                
action_env=TOOLCHAINS_HASH=817bb57d305384a5189ffd1ff11d2f5daabbc1e139d9d9d8021021b8f223ec3b                                                                                                                                                                                                                                                                                                                                      
INFO: Reading 'startup' options from /etc/bazel.bazelrc: --output_user_root=/bazel_root/build_ubuntu_817bb57                                                                                                                                                                                                                                                                                                                     
INFO: Reading 'startup' options from /root/.bazelrc: --output_base=/bazel_root/build_ubuntu_817bb57/ea62ed01e987f38b15f2b7ee37d82cd6                                                                                                                                                                                                                                                                                             
INFO: Options provided by the client:                                                                                                                                                                                                                                                                                                                                                                                            
  Inherited 'common' options: --isatty=1 --terminal_columns=209                                                                                                                                                                                                                                                                                                                                                                  
INFO: Reading rc options for 'build' from /etc/bazel.bazelrc:                                                                                                                                                                                                                                                                                                                                                                    
  'build' options: --action_env=TOOLCHAINS_HASH=817bb57d305384a5189ffd1ff11d2f5daabbc1e139d9d9d8021021b8f223ec3b                                                                                                                                                                                                                                                                                                                 
INFO: Reading rc options for 'build' from /src/workspace/.bazelrc:                                                                                                                                                                                                                                                                                                                                                               
  'build' options: --announce_rc --verbose_failures --client_env=CC=clang --cxxopt=-std=c++17 --host_cxxopt=-std=c++17 --copt=-Werror=return-type --copt=-fno-exceptions --per_file_copt=.*boost.*@-fexceptions --per_file_copt=.*cc/aws/proxy.*@-fexceptions --per_file_copt=.*cc/roma.*@-fexceptions --per_file_copt=.*oneTBB.*@-fexceptions -c opt --output_filter=^//((?!(third_party):).)*$` --color=yes --@io_bazel_rules_d
ocker//transitions:enable=false --workspace_status_command=bash tools/get_workspace_status --copt=-Werror=thread-safety-analysis --@com_google_googleurl//build_config:system_icu=0 --@io_opentelemetry_cpp//api:with_abseil=true                                                                                                                                                                                                
INFO: Analyzed 223 targets (389 packages loaded, 36474 targets configured).                                                                                                                                                                                                                                                                                                                                                      
INFO: Found 223 targets...                                                                                                                                                                                                                                                                                                                                                                                                       
INFO: Elapsed time: 22.289s, Critical Path: 12.97s                                                                                                                                                                                                                                                                                                                                                                               
INFO: 21 processes: 8 internal, 13 processwrapper-sandbox.                                                                                                                                                                                                                                                                                                                                                                       
INFO: Build completed successfully, 21 total actions                                                                                                                                                                                                                                                                                                                                                                             
INFO: Reading 'startup' options from /etc/bazel.bazelrc: --output_user_root=/bazel_root/build_ubuntu_817bb57                                                                                                                                                                                                                                                                                                                     
INFO: Reading 'startup' options from /root/.bazelrc: --output_base=/bazel_root/build_ubuntu_817bb57/ea62ed01e987f38b15f2b7ee37d82cd6                                                                                                                                                                                                                                                                                             
INFO: Options provided by the client:                                                                                                                                                                                                                                                                                                                                                                                            
  Inherited 'common' options: --isatty=1 --terminal_columns=209                                                                                                                                                                                                                                                                                                                                                                  
INFO: Reading rc options for 'test' from /etc/bazel.bazelrc:                                                                                                                                                                                                                                                                                                                                                                     
  Inherited 'build' options: --action_env=TOOLCHAINS_HASH=817bb57d305384a5189ffd1ff11d2f5daabbc1e139d9d9d8021021b8f223ec3b                                                                                                                                                                                                                                                                                                       
INFO: Reading rc options for 'test' from /src/workspace/.bazelrc:                                                                                                                                                                                                                                                                                                                                                                
  Inherited 'build' options: --announce_rc --verbose_failures --client_env=CC=clang --cxxopt=-std=c++17 --host_cxxopt=-std=c++17 --copt=-Werror=return-type --copt=-fno-exceptions --per_file_copt=.*boost.*@-fexceptions --per_file_copt=.*cc/aws/proxy.*@-fexceptions --per_file_copt=.*cc/roma.*@-fexceptions --per_file_copt=.*oneTBB.*@-fexceptions -c opt --output_filter=^//((?!(third_party):).)*$` --color=yes --@io_baz
el_rules_docker//transitions:enable=false --workspace_status_command=bash tools/get_workspace_status --copt=-Werror=thread-safety-analysis --@com_google_googleurl//build_config:system_icu=0 --@io_opentelemetry_cpp//api:with_abseil=true                                                                                                                                                                                      
INFO: Reading rc options for 'test' from /src/workspace/.bazelrc:                                                                                                                                                                                                                                                                                                                                                                
  'test' options: --test_output=errors                                                                                                                                                                                                                                                                                                                                                                                           
INFO: Analyzed 58 targets (16 packages loaded, 135 targets configured).                                                                                                                                                                                                                                                                                                                                                          
INFO: Found 58 test targets...                                                                                                                                                                                                                                                                                                                                                                                                   
                                                                                                                                                                                                                                                                                                                                                                                                                                 
TIMEOUT: //components/data/blob_storage:blob_storage_change_notifier_test (Summary)                                                                                                                                                                                                                                                                                                                                              
      /bazel_root/build_ubuntu_817bb57/ea62ed01e987f38b15f2b7ee37d82cd6/execroot/__main__/bazel-out/k8-opt/testlogs/components/data/blob_storage/blob_storage_change_notifier_test/test.log                                                                                                                                                                                                                                      
INFO: From Testing //components/data/blob_storage:blob_storage_change_notifier_test:                                                                                                                                                                                                                                                                                                                                             
==================== Test output for //components/data/blob_storage:blob_storage_change_notifier_test:                                                                                                                                                                                                                                                                                                                           
Running main() from gmock_main.cc                                                                                                                                                                                                                                                                                                                                                                                                
[==========] Running 10 tests from 1 test suite.                                                                                                                                                                                                                                                                                                                                                                                 
[----------] Global test environment set-up.                                                                                                                                                                                                                                                                                                                                                                                     
[----------] 10 tests from BlobStorageChangeNotifierS3Test                                                                                                                                                                                                                                                                                                                                                                       
[ RUN      ] BlobStorageChangeNotifierS3Test.AwsSqsUnavailable                                                                                                                                                                                                                                                                                                                                                                   
                                                                                                                                                                                                                                                                                                                                                                                                                                 
GMOCK WARNING:                                                                                                                                                                                                                                                                                                                                                                                                                   
Uninteresting mock function call - returning directly.                                                                                                                                                                                                                                                                                                                                                                           
    Function call: RegisterHistogram("AwsSqsReceiveMessageLatency", "AWS SQS ReceiveMessage latency", "microsecond", { 160, 220, 280, 320, 640, 1200, 2500, 5000, 10000, 20000, 40000, 80000, 160000, 320000, 640000, 1e+06, 1.3e+06, 2.6e+06, 5e+06, 10000000000 })                                                                                                                                                             
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.                                                                                                                                  
WARNING: Logging before InitGoogleLogging() is written to STDERR                                                                                                                                                                                                                                                                                                                                                                 
I20230730 06:54:18.468299  3689 change_notifier_aws.cc:100] Getting notifications for topic                                                                                                                                                                                                                                                                                                                                      
E20230730 06:54:24.610873  3689 change_notifier_aws.cc:147] Failed to TagQueue with last_updated: 1690700058 UNKNOWN: Invalid DNS Label found in URI host                                                                                                                                                                                                                                                                        
                                                                                                                                                                                                                                                                                                                                                                                                                                 
GMOCK WARNING:                                                                                                                                                                                                                                                                                                                                                                                                                   
Uninteresting mock function call - returning directly.                                                                                                                                                                                                                                                                                                                                                                           
    Function call: IncrementEventCounter("AwsChangeNotifierTagFailure")                                                                                                                                                                                                                                                                                                                                                          
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.                                                                                                                                  
                                                                                                                                                                                                                                                                                                                                                                                                                                 
GMOCK WARNING:                                                                                                                                                                                                                                                                                                                                                                                                                   
Uninteresting mock function call - returning default value.                                                                                                                                                                                                                                                                                                                                                                      
    Function call: ReceiveMessage(@0x7ffeb64444f0 368-byte object <E8-56 BB-7C AE-7F 00-00 40-5A B1-7C AE-7F 00-00 00-00 00-00 00-00 00-00 30-E1 A7-7C AE-7F 00-00 20-E1 A7-7C AE-7F 00-00 00-00 00-00 D2-00 00-00 00-00 00-00 AE-7F 00-00 00-00 00-00 00-00 00-00 ... 01-00 00-00 00-00 00-00 00-00 00-00 01-00 00-00 01-46 44-B6 FE-7F 00-00 48-46 44-B6 FE-7F 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 60-46 44-B
6 FE-7F 00-00 00-00 00-00 00-00 00-00>)                                                                                                                                                                                                                                                                                                                                                                                          
          Returns: 328-byte object <00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 28-3C 44-B6 FE-7F 00-00 00-00 00-00 00-00 00-00 00-5B 2B-7B AE-7F 00-00 6F-01 00-00 00-00 00-00 00-41 44-B6 FE-7F 00-00 ... 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 01-C5 F5-0F 63-55 00-00 30-3D 44-B6 FE-7F 00-00 00-00 00-00 00-00 00-00 00-40 44-B6 FE-7F 00-00 00-0B B2-FB 1F-F8 09-AC 00-2A C0-0E 63-55
 00-00>                                                                                                                                                                                                                                                                                                                                                                                                                          
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.
E20230730 06:54:24.611037  3689 change_notifier_aws.cc:169] Failed to receive message from SQS: 

GMOCK WARNING:                                                                                          
Uninteresting mock function call - returning directly.
    Function call: IncrementEventCounter("AwsChangeNotifierMessagesReceivingFailure")
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.
I20230730 06:54:24.611056  3689 change_notifier_aws.cc:176] Will create a new Queue

GMOCK WARNING:                                                                                          
Uninteresting mock function call - returning directly.
    Function call: Reset()                                                                              
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.
[       OK ] BlobStorageChangeNotifierS3Test.AwsSqsUnavailable (12509 ms)
[ RUN      ] BlobStorageChangeNotifierS3Test.InvalidJsonMessage

GMOCK WARNING:                                                                                          
Uninteresting mock function call - returning directly.
    Function call: RegisterHistogram("AwsSqsReceiveMessageLatency", "AWS SQS ReceiveMessage latency", "microsecond", { 160, 220, 280, 320, 640, 1200, 2500, 5000, 10000, 20000, 40000, 80000, 160000, 320000, 640000, 1e+06, 1.3e+06, 2.6e+06, 5e+06, 10000000000 })
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.
I20230730 06:54:30.755725  3689 change_notifier_aws.cc:100] Getting notifications for topic 
E20230730 06:54:36.899458  3689 change_notifier_aws.cc:147] Failed to TagQueue with last_updated: 1690700070 UNKNOWN: Invalid DNS Label found in URI host                                                                                                                                                                                                                                                                        

GMOCK WARNING:                                                                                          
Uninteresting mock function call - returning directly.
    Function call: RecordHistogramEvent("AwsSqsReceiveMessageLatency", 25)
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.

GMOCK WARNING:                                                                                          
Uninteresting mock function call - returning default value.
    Function call: DeleteMessageBatch(@0x7ffeb6444950 (incomplete type))
          Returns: 352-byte object <00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 50-3C 44-B6 FE-7F 00-00 00-00 00-00 00-00 00-00 ... 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 01-00 00-00 00-00 00-00 58-3D 44-B6 FE-7F 00-00 00-00 00-00 00-00 00-00 00-0B B2-FB 1F-F8 09-AC 9D-24 C0-0E 63-55 00-00 00-42 44-B6 FE-7F
 00-00>
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.
E20230730 06:54:36.899668  3689 change_notifier_aws.cc:221] Failed to delete message from SQS: 
E20230730 06:54:36.899689  3689 blob_storage_change_notifier_s3.cc:53] Failed to parse JSON. Error: INVALID_ARGUMENT: Failed to parse JSON at: this is not valid json Message:this is not valid json                                                                                                                                                                                                                             
[       OK ] BlobStorageChangeNotifierS3Test.InvalidJsonMessage (12288 ms)
[ RUN      ] BlobStorageChangeNotifierS3Test.JsonHasNoMessageObject

GMOCK WARNING:                                                                                          
Uninteresting mock function call - returning directly.
    Function call: RegisterHistogram("AwsSqsReceiveMessageLatency", "AWS SQS ReceiveMessage latency", "microsecond", { 160, 220, 280, 320, 640, 1200, 2500, 5000, 10000, 20000, 40000, 80000, 160000, 320000, 640000, 1e+06, 1.3e+06, 2.6e+06, 5e+06, 10000000000 })
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.
I20230730 06:54:43.043372  3689 change_notifier_aws.cc:100] Getting notifications for topic 
E20230730 06:54:49.187690  3689 change_notifier_aws.cc:147] Failed to TagQueue with last_updated: 1690700083 UNKNOWN: Invalid DNS Label found in URI host                                                                                                                                                                                                                                                                        

GMOCK WARNING:                                                                                          
Uninteresting mock function call - returning directly.
    Function call: RecordHistogramEvent("AwsSqsReceiveMessageLatency", 24)
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.

GMOCK WARNING:                                                                                          
Uninteresting mock function call - returning default value.
          Returns: 352-byte object <00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 50-3C 44-B6 FE-7F 00-00 00-00 00-00 00-00 00-00 ... 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 01-00 00-00 00-00 00-00 58-3D 44-B6 FE-7F 00-00 00-00 00-00 00-00 00-00 00-0B B2-FB 1F-F8 09-AC 9D-24 C0-0E 63-55 00-00 00-42 44-B6 FE-7F
 00-00>                                                                                                 
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.
E20230730 06:54:49.187901  3689 change_notifier_aws.cc:221] Failed to delete message from SQS:          
E20230730 06:54:49.187924  3689 blob_storage_change_notifier_s3.cc:53] Failed to parse JSON. Error: INVALID_ARGUMENT: Message is not a string Message:{}                                                                                                                                                                                                                                                                         
[       OK ] BlobStorageChangeNotifierS3Test.JsonHasNoMessageObject (12288 ms)                          
[ RUN      ] BlobStorageChangeNotifierS3Test.MessageObjectIsNotAString                                                                                                                                          
                                                                                                                                                                                                                
GMOCK WARNING:                                                                                          
Uninteresting mock function call - returning directly.                                                  
    Function call: RegisterHistogram("AwsSqsReceiveMessageLatency", "AWS SQS ReceiveMessage latency", "microsecond", { 160, 220, 280, 320, 640, 1200, 2500, 5000, 10000, 20000, 40000, 80000, 160000, 320000, 640000, 1e+06, 1.3e+06, 2.6e+06, 5e+06, 10000000000 })
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.
I20230730 06:54:55.332072  3689 change_notifier_aws.cc:100] Getting notifications for topic             
E20230730 06:55:01.475626  3689 change_notifier_aws.cc:147] Failed to TagQueue with last_updated: 1690700095 UNKNOWN: Invalid DNS Label found in URI host                                                                                                                                                                                                                                                                        
                                                                                                                                                                                                                                                                                                                                                                                                                                 
GMOCK WARNING:                                                                                          
Uninteresting mock function call - returning directly.                                                  
    Function call: RecordHistogramEvent("AwsSqsReceiveMessageLatency", 18)                              
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.
                                                                                                        
GMOCK WARNING:                                                                                          
Uninteresting mock function call - returning default value.                                                                                                                                                     
    Function call: DeleteMessageBatch(@0x7ffeb6444950 (incomplete type))                                
          Returns: 352-byte object <00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 50-3C 44-B6 FE-7F 00-00 00-00 00-00 00-00 00-00 ... 00-00 00-00 00-00 00-00 00-00 00-00 00-00 00-00 01-00 00-00 00-00 00-00 58-3D 44-B6 FE-7F 00-00 00-00 00-00 00-00 00-00 00-0B B2-FB 1F-F8 09-AC 9D-24 C0-0E 63-55 00-00 00-42 44-B6 FE-7F
 00-00>                                                                                                 
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.
E20230730 06:55:01.475795  3689 change_notifier_aws.cc:221] Failed to delete message from SQS:          
E20230730 06:55:01.475816  3689 blob_storage_change_notifier_s3.cc:53] Failed to parse JSON. Error: INVALID_ARGUMENT: Message is not a string Message:{                                                                                                                                                                                                                                                                          
   "Message": {}                                                                                        
  }                                                                                                     
[       OK ] BlobStorageChangeNotifierS3Test.MessageObjectIsNotAString (12287 ms)                       
[ RUN      ] BlobStorageChangeNotifierS3Test.RecordsIsNotAList                                          
                                                                                                                                                                                                                                                                                                                                                                                                                                 
GMOCK WARNING:                                                                                                                                                                                                                                                                                                                                                                                                                   
Uninteresting mock function call - returning directly.                                                  
    Function call: RegisterHistogram("AwsSqsReceiveMessageLatency", "AWS SQS ReceiveMessage latency", "microsecond", { 160, 220, 280, 320, 640, 1200, 2500, 5000, 10000, 20000, 40000, 80000, 160000, 320000, 640000, 1e+06, 1.3e+06, 2.6e+06, 5e+06, 10000000000 })
NOTE: You can safely ignore the above warning unless this call should not happen.  Do not suppress it by blindly adding an EXPECT_CALL() if you don't mean to enforce the call.  See https://github.com/google/googletest/blob/main/docs/gmock_cook_book.md#knowing-when-to-expect for details.
I20230730 06:55:07.618567  3689 change_notifier_aws.cc:100] Getting notifications for topic             
-- Test timed out at 2023-07-30 06:55:11 UTC --                                                         
================================================================================  
INFO: Elapsed time: 62.775s, Critical Path: 60.06s                                                      
INFO: 60 processes: 2 local, 65 processwrapper-sandbox.                                                                                                                                                                                                                                                                                                                                                                          
INFO: Build completed, 1 test FAILED, 60 total actions                                                  
//components/aws:sqs_cleanup_manager_test                                PASSED in 0.2s                 
//components/cloud_config:parameter_client_test                          PASSED in 37.1s                
//components/data/blob_storage:blob_storage_client_test                  PASSED in 30.9s                                                                                                                                                                                                                                                                                                                                         
//components/data/blob_storage:delta_file_notifier_test                  PASSED in 0.2s                 
//components/data/common:change_notifier_test                            PASSED in 43.2s                                                                                                                                                                                                                                                                                                                                         
//components/data/realtime:delta_file_record_change_notifier_test        PASSED in 0.3s                 
//components/data/realtime:realtime_notifier_test                        PASSED in 0.2s                                                                                                                                                                                                                                                                                                                                          
//components/data_server/cache:key_value_cache_test                      PASSED in 0.1s                 
//components/data_server/data_loading:data_orchestrator_test             PASSED in 0.4s                 
//components/data_server/request_handler:get_values_adapter_test         PASSED in 0.3s                 
//components/data_server/request_handler:get_values_handler_test         PASSED in 0.3s                 
//components/data_server/request_handler:get_values_v2_handler_test      PASSED in 0.2s                 
//components/data_server/request_handler:ohttp_encryptor_test            PASSED in 0.1s                                                                                                                                                                                                                                                                                                                                          
//components/data_server/request_handler:v2_response_data_proto_test     PASSED in 0.1s                                                                                                                                                                                                                                                                                                                                          
//components/data_server/server:lifecycle_heartbeat_test                 PASSED in 0.3s                 
//components/data_server/server:parameter_fetcher_test                   PASSED in 0.3s                                                                                                                                                                                                                                                                                                                                          
//components/data_server/server:server_buildinfo_test                    PASSED in 0.0s                 
//components/data_server/server:server_lib_test                          PASSED in 0.5s                 
//components/errors:error_util_aws_test                                  PASSED in 0.0s                                                                                                                                                                                                                                                                                                                                          
//components/errors:retry_test                                           PASSED in 0.2s
//components/internal_server:lookup_server_impl_test                     PASSED in 0.3s
//components/internal_server:remote_lookup_client_impl_test              PASSED in 0.2s
//components/internal_server:sharded_lookup_server_impl_test             PASSED in 0.3s
//components/internal_server:string_padder_test                          PASSED in 0.1s
//components/query:ast_test                                              PASSED in 0.0s
//components/query:driver_test                                           PASSED in 0.0s
//components/query:scanner_test                                          PASSED in 0.0s
//components/sharding:cluster_mappings_manager_test                      PASSED in 2.3s
//components/sharding:shard_manager_test                                 PASSED in 0.3s
//components/tools/benchmarks:benchmark_util_test                        PASSED in 0.1s
//components/udf:cache_get_values_hook_test                              PASSED in 0.3s
//components/udf:get_values_hook_test                                    PASSED in 0.2s
//components/udf:run_query_hook_test                                     PASSED in 0.2s
//components/udf:udf_client_test                                         PASSED in 0.6s
//components/util:periodic_closure_test                                  PASSED in 0.1s
//components/util:sleepfor_test                                          PASSED in 4.0s
//production/packaging/aws/data_server:commands_test                     PASSED in 1.1s
//public:base_types_proto_lint                                           PASSED in 0.0s
//public:constants_test                                                  PASSED in 0.0s
//public/data_loading:filename_utils_test                                PASSED in 0.0s
//public/data_loading:records_utils_test                                 PASSED in 0.1s
//public/data_loading:riegeli_metadata_proto_lint                        PASSED in 0.1s
//public/data_loading/csv:csv_delta_record_stream_reader_test            PASSED in 0.0s
//public/data_loading/csv:csv_delta_record_stream_writer_test            PASSED in 0.0s
//public/data_loading/readers:delta_record_stream_reader_test            PASSED in 0.1s
//public/data_loading/readers:riegeli_stream_io_test                     PASSED in 2.5s
//public/data_loading/writers:delta_record_stream_writer_test            PASSED in 0.1s
//public/query:get_values_proto_lint                                     PASSED in 0.1s
//public/query/v2:get_values_proto_lint                                  PASSED in 0.1s
//public/sharding:sharding_function_test                                 PASSED in 0.0s
//tools/bidding_auction_data_generator:custom_audience_data_parser_test  PASSED in 0.1s
//tools/bidding_auction_data_generator:delta_key_value_writer_test       PASSED in 0.1s
//tools/bidding_auction_data_generator:http_value_retriever_test         PASSED in 0.6s
//tools/data_cli/commands:format_data_command_test                       PASSED in 0.1s
//tools/request_simulation:message_queue_test                            PASSED in 0.0s
//tools/request_simulation:rate_limiter_test                             PASSED in 0.0s
//tools/request_simulation:request_generation_util_test                  PASSED in 0.1s
//components/data/blob_storage:blob_storage_change_notifier_test        TIMEOUT in 60.0s
  /bazel_root/build_ubuntu_817bb57/ea62ed01e987f38b15f2b7ee37d82cd6/execroot/__main__/bazel-out/k8-opt/testlogs/components/data/blob_storage/blob_storage_change_notifier_test/test.log                                                                                                                                                                                                                                          

INFO: Build completed, 1 test FAILED, 60 total actions
Collecting bazel logs ['aws-aws-logs.zip']... [status: 0]
INFO: Reading 'startup' options from /etc/bazel.bazelrc: --output_user_root=/bazel_root/build_ubuntu_817bb57                                                                                                                                                                                                                                                                                                                     
INFO: Reading 'startup' options from /root/.bazelrc: --output_base=/bazel_root/build_ubuntu_817bb57/ea62ed01e987f38b15f2b7ee37d82cd6                                                                                                                                                                                                                                                                                             
INFO: Options provided by the client:                                                                   
  Inherited 'common' options: --isatty=1 --terminal_columns=209
INFO: Reading rc options for 'run' from /etc/bazel.bazelrc:
  Inherited 'build' options: --action_env=TOOLCHAINS_HASH=817bb57d305384a5189ffd1ff11d2f5daabbc1e139d9d9d8021021b8f223ec3b                                                                                                                                                                                                                                                                                                       
INFO: Reading rc options for 'run' from /src/workspace/.bazelrc:
  Inherited 'build' options: --announce_rc --verbose_failures --client_env=CC=clang --cxxopt=-std=c++17 --host_cxxopt=-std=c++17 --copt=-Werror=return-type --copt=-fno-exceptions --per_file_copt=.*boost.*@-fexceptions --per_file_copt=.*cc/aws/proxy.*@-fexceptions --per_file_copt=.*cc/roma.*@-fexceptions --per_file_copt=.*oneTBB.*@-fexceptions -c opt --output_filter=^//((?!(third_party):).)*$` --color=yes --@io_baz
el_rules_docker//transitions:enable=false --workspace_status_command=bash tools/get_workspace_status --copt=-Werror=thread-safety-analysis --@com_google_googleurl//build_config:system_icu=0 --@io_opentelemetry_cpp//api:with_abseil=true
INFO: Analyzed target //:collect-logs (0 packages loaded, 1 target configured).
INFO: Found 1 target...                                                                                 
Target //:collect-logs up-to-date:                                                                      
  bazel-bin/collect_logs.bin                                                                            
INFO: Elapsed time: 0.851s, Critical Path: 0.01s                                                        
INFO: 1 process: 1 internal.                                                                            
INFO: Build completed successfully, 1 total action                                                      
INFO: Build completed successfully, 1 total action                                                      
stored bazel logs to /src/workspace/dist/aws-aws-logs.zip
Archive:  /src/workspace/dist/aws-aws-logs.zip                                                          
Zip file size: 113476 bytes, number of entries: 229                                                     

build_and_test_all_in_docker runtime: 100s                                                              
Error: build_and_test_in_docker completed with status code: 3


[ERROR]: Failure: Failed to run build_and_test_all_in_docker


aws/build_and_test_all runtime: 105s                                                                    
Error: aws/build_and_test completed with status code: 1
```

This is a log file output in `/bazel_root/build_ubuntu_817bb57/ea62ed01e987f38b15f2b7ee37d82cd6/execroot/__main__/bazel-out/k8-opt/testlogs/components/data/blob_storage/blob_storage_change_notifier_test/test.log`. 
[build_storage_change_notifier_test.log](https://github.com/privacysandbox/fledge-key-value-service/files/12208670/build_storage_change_notifier_test.log)

**Comment by peiwenhu** [comment]
Thanks for reporting this. Let us try to reproduce it on our end

**Comment by peiwenhu** [comment]
Hi @k-o-ta , we've released a new patch 0.11.1 that disables the tests. It should have this issue resolved. Please let me know if that's not the case. thanks

**Comment by k-o-ta** [comment]
I confirmed that v0.11.1 resolves this problem.
Thanks!
